ui/ui-components/components/appSelectorList.tsx

```diff
@@ -9,75 +37,107 @@ export default function AppSelectorList({
   displayAddAppButton = false,
 }: {
   onClick: Function;
-  items: Array<any>;
+  items:
+    | SelectableApp[]
+    | SelectablePackage[]
+    | SelectableSource[]
+    | SelectableDestination[];
```

Comment:
```suggestion
  items:
    (SelectableApp
    | SelectablePackage
    | SelectableSource
    | SelectableDestination)[];
```
Maybe?

Reply:
good catch! It's an array of one type of thing - it won't be mixed